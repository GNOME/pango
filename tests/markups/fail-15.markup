<span style="funny">test</span>
