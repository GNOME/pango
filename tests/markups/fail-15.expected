ERROR: 'funny' is not a valid value for the 'style' attribute on <span> tag, line 1; valid values are 'normal', 'oblique', 'italic'